src/containers/home/index.tsx

```diff
@@ -28,24 +28,42 @@ import {
 const { Text, Paragraph } = Typography;
 
 const LandingContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+`;
+
+const LandingImage = styled.div`
   width: 100%;
   height: calc(95vh - 110px);
   object-fit: cover;
   background-image: url('${HOME_IMAGE}');
   background-position: center; /* Center the image */
   background-repeat: no-repeat; /* Do not repeat the image */
   background-size: cover; /* Resize the background image to cover the entire container */
+
+  @media screen and (max-width: 900px) {
```

Comment:
⛏️ 900px should be extracted as a constant so that we can change breakpoints easily later